frontend-react/e2e/spec/all/public/developer-resources/api/documentation/index.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { DeveloperResourcesApiDocumentationPage } from "../../../../../../pages/public/developer-resources/api/documentation/index";
+import { test as baseTest, expect } from "../../../../../../test";
+
+export interface SecurityPageFixtures {
+    developerResourcesApiDocumentationPage: DeveloperResourcesApiDocumentationPage;
+}
+
+const test = baseTest.extend<SecurityPageFixtures>({
+    developerResourcesApiDocumentationPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new DeveloperResourcesApiDocumentationPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "Developer Resources / API / Documentation / Index page",
+    {
+        tag: "@smoke",
+    },
+    () => {
+        test("has correct title", async ({ developerResourcesApiDocumentationPage }) => {
```

Comment:
You do not need this section since this code below already tests for that:

```
test("has correct title + heading", async ({ developerResourcesApiDocumentationPage }) => {
            await developerResourcesApiDocumentationPage.testHeader();
        });
```

Reply:
This applies to `frontend-react/e2e/spec/all/public/developer-resources/api/documentation/responses-from-reportstream.spec.ts`, `frontend-react/e2e/spec/all/public/developer-resources/api/documentation/sample-payloads-and-output.spec.ts` and `frontend-react/e2e/spec/all/public/developer-resources/api/getting-started.spec.ts` as well.